src/components/common/Extensions/UserNavigation/partials/HeaderAvatar/HeaderAvatar.tsx

```diff
@@ -16,7 +16,7 @@ const HeaderAvatar = () => {
 
   return (
     <div className="flex items-center justify-center min-w-[2.625rem] min-h-[2.5rem] px-[0.875rem] py-[0.625rem] bg-base-white border rounded-full border-gray-200">
-      <UserAvatar user={user || wallet?.address} showUsername size="xxs" />
+      <UserAvatar user={user || wallet?.address} showUsername size="xss" />
```

Comment:
@CzarekDryl This change broke the user name display on the landing page

![Screenshot from 2024-02-15 17-40-30](https://github.com/JoinColony/colonyCDapp/assets/1193222/52d584d9-ddb1-43d8-8667-fe68bb56539b)
![Screenshot from 2024-02-15 17-44-41](https://github.com/JoinColony/colonyCDapp/assets/1193222/2bacbc52-8931-4d6a-80ed-1b1d0c54d1ae)

However, it does work fine when you're on a colony's home / dashboard

![Screenshot from 2024-02-15 17-40-35](https://github.com/JoinColony/colonyCDapp/assets/1193222/9faeb4a9-f9d5-459b-9e5f-a18c937db245)

Please open a separate PR to fix this.